inc/classes/traits/trait-updater-api.php

```diff
@@ -0,0 +1,97 @@
+<?php
+namespace WP_Rocket\Traits;
+
+use WP_Rocket\Logger\Logger;
+
+/**
+ * Trait for the plugin updater.
+ *
+ * @since  3.3.6
+ * @author Grégory Viguier
+ */
+trait Updater_Api_Subscriber {
+	/**
+	 * An ID to use when a API request fails.
+	 *
+	 * @var    string
+	 * @since  3.3.6
+	 * @access protected
+	 * @author Grégory Viguier
+	 */
+	/*protected $request_error_id;*/
```

Comment:
It can be removed instead of commented

Reply:
Imho it is better to keep it to document it, and prevent forgetting it.

Reply:
Fair enough 👍